src/Resources/views/datatable_js.html.twig

```diff
@@ -144,67 +53,5 @@
         {% endfor %}
         {% endif %}
 
-        oTable.DataTable(defaults);
     });
-
-    (function ($) {
-        $.deparam = function (params, coerce) {
-            var obj = {},
-                coerce_types = {'true': !0, 'false': !1, 'null': null};
-            $.each(params.replace(/\+/g, ' ').split('&'), function (j, v) {
-                var param = v.split('='),
-                    key = decodeURIComponent(param[0]),
-                    val,
-                    cur = obj,
-                    i = 0,
-                    keys = key.split(']['),
-                    keys_last = keys.length - 1;
-
-                if (/\[/.test(keys[0]) && /\]$/.test(keys[keys_last])) {
-                    keys[keys_last] = keys[keys_last].replace(/\]$/, '');
-                    keys = keys.shift().split('[').concat(keys);
-                    keys_last = keys.length - 1;
-                } else {
-                    keys_last = 0;
-                }
-
-                if (param.length === 2) {
-                    val = decodeURIComponent(param[1]);
-
-                    if (coerce) {
-                        val = val && !isNaN(val) ? +val              // number
-                            : val === 'undefined' ? undefined         // undefined
-                                : coerce_types[val] !== undefined ? coerce_types[val] // true, false, null
-                                    : val;                                                // string
-                    }
-
-                    if (keys_last) {
-                        for (; i <= keys_last; i++) {
-                            key = keys[i] === '' ? cur.length : keys[i];
-                            cur = cur[key] = i < keys_last
-                                ? cur[key] || (keys[i + 1] && isNaN(keys[i + 1]) ? {} : [])
-                                : val;
-                        }
-
-                    } else {
-                        if ($.isArray(obj[key])) {
-                            obj[key].push(val);
-                        } else if (obj[key] !== undefined) {
-                            obj[key] = [obj[key], val];
-                        } else {
-                            obj[key] = val;
-                        }
-                    }
-
-                } else if (key) {
-                    obj[key] = coerce
-                        ? undefined
-                        : '';
-                }
-            });
-
-            return obj;
-        };
-    })(jQuery);
 </script>
```

Comment:
I am not clear on the relationship between this script and the non templated version in that is installed on assets:install, only that the other does not provide search/filter behaviors.

Reply:
I'm pretty sure we left this old code in here as reference, as I was aware I was breaking this functionality. The refactor itself was actually mostly about relocating inline generated JS to the separate asset file. The underlying "controversy" is mostly how much support we want to give for optional DT extensions.